src/main/scala/mesosphere/marathon/raml/DefaultConversions.scala

```diff
@@ -60,4 +63,9 @@ trait DefaultConversions {
       .getOrElse(throw new IllegalArgumentException(s"$name is an unknown protocol"))
     IpAddr(ipAddress.getIpAddress, protocol)
   }
+
+  implicit object OffsetDateTimeWrite extends play.api.libs.json.Writes[OffsetDateTime] {
+    def writes(o: OffsetDateTime) = JsString(o.format(Timestamp.formatter))
```

Comment:
What about reads? Or is this only used for API models?

Reply:
Only for the API. I want to keep `Reads` default since they're not strict and therefore backwards compatible.